packages/core/labels.scss

```diff
@@ -4,17 +4,22 @@
 
 :root,
 [data-theme="light"] {
+    --label-color: #{$granitt};
+    --label-support-color: #{$stein};
     --label-error-color: #{$feil};
 }
 
 [data-theme="dark"] {
+    --label-color: #{$snohvit};
+    --label-support-color: #{$svaberg};
     --label-error-color: #{$feil--darkbg};
 }
 
 .jkl-form-support-label {
     @include jkl-text-style("desktop/small");
     display: block;
     margin-top: $component-spacing--small;
+    color: var(--label-support-color);
```

Comment:
Har vi glemt å assigne farge til variabel først? Flere andre steder har man brukt. Usikker på hvorfor, må vel være fordi IE ikke støtter CSS-variabler eller noe sånt, usikker på om vi fortsatt prøver å sørge for IE11-støtte i Jøkul?
```
color: $variable-name
color: var(--variable-name)
```

Reply:
Det var slik den var fra før av og tipper at det er pga IE11

Reply:
Jeg gjorde som de andre filene nå